lib/commands/addValue.js

```diff
@@ -56,7 +56,7 @@ let addValue = function (selector, value) {
         let elementIdValueCommands = []
 
         for (let elem of res.value) {
-            elementIdValueCommands.push(self.elementIdValue(elem.ELEMENT, value))
+            elementIdValueCommands.push(self.elementIdValue(elem[Object.keys(elem)[0]], value))
```

Comment:
Checked the code again and found, that we add elementId in path in protocol helpers. Looks like we shouldn't move this logic to request handler. I propose to create new helper in utils that will return elementId. and use it in each command helper